spec/system/add_comment_spec.rb

```diff
@@ -57,9 +69,9 @@
 
     describe 'viewing comments' do
       it 'shows comments under the idea' do
-        idea.comments.create!(body: 'Comment 1', user: default_user)
-        idea.comments.create!(body: 'Comment 2', user: default_user)
-        visit idea_path(idea)
+        approved_idea.comments.create!(body: 'Comment 1', user: default_user)
```

Comment:
Probably worth using factories for this? :)

Reply:
I'll create a new PR and card :)